flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemConnectorOptions.java

```diff
@@ -245,6 +245,13 @@ public class FileSystemConnectorOptions {
                     .withDescription(
                             "The compaction target file size, the default value is the rolling file size.");
 
+    public static final ConfigOption<Boolean> HIVE_SINK_PARTITION_COMMIT_SUCCESS_FILE_ENABLED =
+            key("hive.sink.partition-commit.success-file.enabled")
```

Comment:
Do we really need this configuration? I think this can be covered by `sink.partition-commit.policy.kind=success-file`.